cmd/kops/create_cluster.go

```diff
@@ -300,7 +299,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.RegisterFlagCompletionFunc("subnets", completeSubnetID(options))
 	cmd.Flags().StringSliceVar(&options.UtilitySubnetIDs, "utility-subnets", options.UtilitySubnetIDs, "Shared utility subnets to use")
 	cmd.RegisterFlagCompletionFunc("utility-subnets", completeSubnetID(options))
-	cmd.Flags().StringVar(&options.NetworkCIDR, "network-cidr", options.NetworkCIDR, "Network CIDR to use")
+	cmd.Flags().StringSliceVar(&options.NetworkCIDRs, "network-cidr", options.NetworkCIDRs, "Network CIDR(s) to use")
```

Comment:
My suggestion would be to add a new flag called `additional-network-cidrs` to be backward compatible.

Reply:
It is already backwards compatible. Nothing changes from user point of view.

Reply:
Sorry, might be missing something here but won't going from `StringVar` to `StringSliceVar` require users to make a change to pass a slice of strings now?

Reply:
you can pass a slice as multiple `--network-cidr` flags or comma separated.
For example, with this change, one can just do:
```
--network-cidr=10.0.0.0/16 --network-cidr=10.1.0.0/16 --network-cidr=10.2.0.0/16 --network-cidr=10.3.0.0/16 --network-cidr=10.4.0.0/16 --network-cidr=10.5.0.0/16
```

Reply:
Right, it's a nice feature of the flags library!